### PR TITLE
Added description and priority order for Layered Profile in server's form.

### PR DESCRIPTION
**Pull request by rimashah25**
Added helpful description to understand layered profile
Relates: https://github.com/apache/trafficcontrol/pull/6095

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?

View servers's page (either create new server or click on existing server and one should see a note explaing layered profile and how it works and priority order.

Example:
image

## PR submission checklist
- [ ] This PR has tests 
- [ ] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_portal/app/src/common/modules/form/server/form.server.tpl.html [hunk 1]
```diff
@@ -111,8 +111,18 @@
             </div>
 
             <fieldset>
-                <legend>Profiles<button name="addProfileBtn" class="btn btn-primary right-button btn-xs" type="button" title="add a new profile to the server" ng-click="addProfile()"><i class="fa fa-plus"></i></button></legend>
-                <fieldset ng-repeat="profile in server.profileNames track by $index">
+                <legend title="Layered Profiles for a given server."> Profiles
+                    <button name="addProfileBtn" class="btn btn-primary right-button btn-xs" type="button" title="add new profile(s) to the server" ng-click="addProfile()"><i class="fa fa-plus"></i></button>
+                </legend>
+                <div class="helptext">
```
Comment (ocket8888) [inline traffic_portal/app/src/common/modules/form/server/form.server.tpl.html:117]: This `DIV` doesn't seem to be serving any purpose. It isn't displaying like other `.helptext` elements do - i.e. on hover - and it doesn't appear to be styled in any way so CSS isn't using it. It's just a container element with no semantic meaning, so it's not important to the document structure.

So, if this isn't meant to be on-hover help text, I'd suggest removing the div so nobody seeing that thinks it's broken. If it *is* meant to be on-hover help text, then some work needs to be done to get that to display properly.

Comment (rimashah25) [reply traffic_portal/app/src/common/modules/form/server/form.server.tpl.html:117]: It is not on-hover help test, so I will remove `<div>`

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [unreleased]
 ### Added
+- *Traffic Portal* Added description and priority order for Layered Profile on server form.
```
[the remark this answers is not in the document]
Comment (rimashah25) [reply CHANGELOG.md:8]: But there isn't a GH issue for the same. So where do I get a link for it?

[the remark this answers is not in the document]
Comment (ocket8888) [reply CHANGELOG.md:8]: In other cases like that, the link is to the PR

**Review by ocket8888 — APPROVED** [review]

**Pull request merged**